### ISSUE TITLE
Conduct assessment of data in the portal vs. etenders website

### ISSUE THREAD

**Issue by bezunakarmi** [younginnovations/opencontracting-moldova#6]
Conduct assessment of data in the portal vs. etenders website and validate proper availability of data in the portal.

- [ ] Need to share the process to PPA for sustainability
- [ ] Share an assessment done until March 2017 maybe (so that from now on PPA only does assessment of new data and has clear overview of past data imports)
- [ ] We need to share the assessment as checklist (eg. the latest date, data points, etc.) so that PPA 

cc @dc-coder

**Comment by anjesh** [comment]
Auto-generating [summary.xlsx](https://github.com/younginnovations/opencontracting-moldova/files/877905/summary.xlsx) using script.

**Comment by mireille-raad** [comment]
can the script randomly pull 10 records from e-tenders json and provide the link to the 10 individual pages on the open contracting website?. The user can then randomly/manually verify that records exist and that details are the same.

I feel this excel sheet is more of a summary of current platform... we need a way to have some comparisons as well.

**Comment by anjesh** [comment]
Yes, it's a summary of the current platform, to ensure that the data we are showing in the portal is correct when the same data is pulled using external queries. We identified few issues from this analysis and fixed them. Now both are correct.

I have created a new summary with the above summary as well as 10 random tenders and contracts from the json file from etenders. Have a look. @dc-coder 

Everytime the script is run, it gets 10 random items from the db. 

[summary.xlsx](https://github.com/younginnovations/opencontracting-moldova/files/902259/summary.xlsx)

**Comment by mireille-raad** [comment]
excellent! I am glad that you found doing the assessment useful to improve code quality.

For the random tenders and contracts from etenders, do you think you can kinda try to pull the same records from the OCDS data using the id and see for example if procuring agency and the total match. If they match, a cell that says "ok" or "record verified" in summary.xls is added? 

What do you think? a good approach or is the verification better be done manually?

**Comment by anjesh** [comment]
I would go for manual verification. Since the data is coming from the same database (containing both etenders json and ocds_release json), verification should normally be ok as we are mapping the etenders json -> ocds. 

I am thinking of including the URL as well. For tenders data, you may use the tender id to get the urls for tender data in both the system
http://etender.gov.md/proceduricard?pid=2385402 
http://moldova-demo.yipl.com.np/tenders/2385402

And with contracts, you have to search in http://etender.gov.md/contracte, which is tricky.

**Comment by anjesh** [comment]
We will update the assessment process (showing private data) once API is ready.

**Comment by anjesh** [comment]
I am closing this as we have completed the assessment (which doesn't include few private fields for now). Will create a new issue once those fields are available.
